recipes-graphics/drm/libdrm_%.bbappend

```diff
@@ -1,5 +1,8 @@
 FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
 
-SRC_URI_append_imxgpu2d = " file://drm-update-arm.patch"
+SRC_URI_GPU_mx6 = "file://drm-update-arm.patch"
+SRC_URI_GPU_mx7 = "file://drm-update-arm.patch"
+SRC_URI_GPU_mx8 = ""
+SRC_URI_append_imxgpu = " ${SRC_URI_GPU}"
```

Comment:
Sorry, I suspect my testing was over too many variables, not exactly sure of all the cases I ran into.

I did see an error because there is no patch file in an mx8 folder (like mx6 and mx7 case). I can add the folder and file, but then the patch is applied with fuzz. When you check it, it applies elif __arm__ block identical to the one already in the header.

That said, it seems that SRC_URI_remove from our fork recipe is working now. I will drop this change and only update our fork recipe.